### ISSUE TITLE
Header button inconsistency 

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6677]
Two type of buttons with same appearance could make the user confusing better to add a down arrow to user dropdown to show it a dropdown. Button hover state is not visible too.

Heuristic evaluation: Consistency and standards

![image](https://user-images.githubusercontent.com/20179540/67547520-f0a4fc00-f71c-11e9-9ea8-2388ffc417d3.png)
